src/components/button/Button.jsx

```diff
@@ -0,0 +1,15 @@
+import './Button.css'
+function Button({buttonType = "button", text, disabled, clickHandler}) {
+    return(
+        <>
+            <button type={buttonType}
+                    disabled={disabled}
+                    onClick={clickHandler}
+            >
+                {text}
+            </button>
+        </>
+    )
+}
```

Comment:
Dit component voelt nu als een doorgeefluik. Wat ik daarmee bedoel te zeggen is dat je net zo goed direct het `button`-element kunnen gebruiken. Dit component zoals die nu is zorgt het niet voor code reductie